docs/advanced_usage.rst

```diff
@@ -488,16 +489,29 @@ the console, and changes can be made as needed.
 uWSGI
 -----
 
-The default configuration of uWSGI applications does not include the
-``--enable-threads`` setting which must be set to ``true`` for the
-tracing library to run.  This is noted in their best practices doc_.
+The tracer and profiler support uWSGI when configured with the following:
+
+- Threads must be enabled.
+- Multiple processes requires `lazy-app <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#lazy-apps>`_ enabled or a `master <https://uwsgi-docs.readthedocs.io/en/latest/Options.html#master>`_ process enabled.
+
+Tracing is not supported when using ``ddtrace-run`` with uWSGI as the tracer needs to be initialized in the worker processes. Use manual configuration instead to enable tracing by adding :ref:`patch_all()<patch_all>` and :ref:`agent configuration<agentconfiguration>` to a WSGI app::
```

Comment:
We should be able to detect this within `ddtrace-run` and inform the user that there may be issues. It's highly unlikely that people are going to see this documentation before trying it.